conda_build/verify.py

```diff
@@ -0,0 +1,44 @@
+import os
+from subprocess import Popen, PIPE
+from .exceptions import VerifyError
+
+
+def get_exitcode_stdout_stderr(cmd):
+    proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
+    out, err = proc.communicate()
+    exitcode = proc.returncode
+    return exitcode, out, err
+
+
+def is_valid_script(script):
+    if os.path.isfile(script) and script.split(".")[-1] == "py":
+        return True
+    return False
+
+
+def run_scripts(verify_scripts, verify_path, *args):
+    for script in verify_scripts:
+        script_path = os.path.join(verify_path, script)
+        print("Runinning script %s ..." % script)
+        cmd = ["python", script_path]
+        cmd.extend(list(args))
+        exitcode, out, err = get_exitcode_stdout_stderr(cmd)
+        if exitcode == 1:
+            raise VerifyError(str(err, 'utf-8'), script)
+
+
+def verify(verify_path, *args):
+    verify_scripts = [f for f in os.listdir(verify_path)
+                      if is_valid_script(os.path.join(verify_path, f))]
+    run_scripts(verify_scripts, verify_path, *args)
+    print("All scripts passed")
+
+
+def verify_package(path_to_package, config):
+    verify_path = os.path.join(config.verify_scripts_path, "package")
+    verify(verify_path, path_to_package)
+
+
+def verify_recipe(recipe, config):
```

Comment:
I think the tricky thing is that for ```__import__``` to work, things must be locatable with python's path.  That means that things probably need to be part of a package that is installed on the system.  Do you already have a package for these scripts, or not?